### ISSUE TITLE
OptiX headers (optix.h and friends) not found.

### ISSUE THREAD

**Issue by bipul-mohanto** [ingowald/optix7course#29]
Hello,

I am using the following configuration:

* `Ubuntu 20.04.4 LTS`
*  `cmake version 3.16.3`
*  Cuda compilation tools, release 11.6, V11.6.124
*  Build cuda_11.6.r11.6/compiler.31057947_0
*  gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
*  gdb 9.2
* NVIDIA-SMI 510.47.03    Driver Version: 510.47.03    CUDA Version: 11.6

During installing OptiX 7.4, I have defined the OptiX_DIR with the command: `export OptiX_INSTALL_DIR=/home/computer-name/Downloads/NVIDIA-OptiX-SDK-7.4.0-linux64-x86_64`

**Problem:**
* Although I have predefined `OptiX_INSTALL_DIR`, if I BUILD `optix7course` from CMakeGUI, I need to define manually again the `OptiX_INSTALL_DIR`. If I set it to `NVIDIA-OptiX-SDK-7.4.0-linux64-x86_64` directory, both configure and generate works fine. However, when I am opening the repository with `CLion IDE`, it is returning the errors:
```
CMake Error at common/gdt/cmake/FindOptiX.cmake:103 (message):
  OptiX headers (optix.h and friends) not found.  Please locate before
  proceeding.
Call Stack (most recent call first):
  common/gdt/cmake/FindOptiX.cmake:115 (OptiX_report_error)
  common/gdt/cmake/configure_optix.cmake:24 (find_package)
  CMakeLists.txt:37 (include)
```

* If I try to build to repository using `cmake ..` from the terminal, the exact same error is showed up. 
* The `CmakeError.log` shows something like this:
```
CheckFunctionExists.c.o -o cmTC_412b3  -lpthreads 
/usr/bin/ld: cannot find -lpthreads
collect2: error: ld returned 1 exit status
make[1]: *** [CMakeFiles/cmTC_412b3.dir/build.make:99: cmTC_412b3] Error 1
make[1]: Leaving directory '/home/computer-name/Desktop/optix7course/BUILD/CMakeFiles/CMakeTmp'
make: *** [Makefile:127: cmTC_412b3/fast] Error 2
```

**Comment by bipul-mohanto** [comment]
![Screenshot from 2022-04-27 12-06-36](https://user-images.githubusercontent.com/12484581/165494910-d11d95b1-55bf-4b13-a77a-57010e518765.png)

```javascript

CMake Error at common/gdt/cmake/FindOptiX.cmake:103 (message):
  OptiX headers (optix.h and friends) not found.  Please locate before
  proceeding.
Call Stack (most recent call first):
  common/gdt/cmake/FindOptiX.cmake:115 (OptiX_report_error)
  common/gdt/cmake/configure_optix.cmake:24 (find_package)
  CMakeLists.txt:37 (include)
```

**Comment by bipul-mohanto** [comment]
Okay, I found the solution. 

1. Update the FindOptiX.cmake with `set(OptiX_INSTALL_DIR "/home/name-computer/Downloads/NVIDIA-OptiX-SDK-7.4.0-linux64-x86_64")`. This solved the first problem, but lead to another `BIN2CC` not found, although I have defined bin2cc in cmake build. 
2. This [previous issue](https://github.com/ingowald/optix7course/issues/28) helped me, I have manually modified the `CMakeCache.txt` and `configure_optix.cmake` with `set(BIN2C /usr/local/cuda-11.6/bin/bin2c)`. This solved the `BIN2CC` problem, but lead me to another compile time error. 
3. So finally I deleted all the `CMakeCache.txt` files, but kept the `configure_optix.cmake` with `set(BIN2C /usr/local/cuda-11.6/bin/bin2c)` and made a fresh cmake build. 
4. And finally it is working.
